.github/workflows/ci.yml

```diff
@@ -7,11 +7,21 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        gemfile: [rails_4.0.0.gemfile, rails_5.0.0.gemfile, rails_5.2.3.gemfile, rails_6.0.0.gemfile, rails_master.gemfile]
-        ruby_version: [2.4.x, 2.5.x, 2.6.x, 2.7.x]
+        gemfile: [rails_4.0.0.gemfile, rails_5.0.0.gemfile, rails_5.2.3.gemfile, rails_6.0.0.gemfile, rails_7.0.0.gemfile, rails_master.gemfile]
+        ruby_version: [2.4, 2.5, 2.6, 2.7, 3.0]
```

Comment:
The CI is still failing, probably it's related to the deprecation of `actions/setup-ruby` in favor of `ruby/setup-ruby`. We might need to change it in line 30 https://github.com/rootstrap/arqo/pull/17/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR30